draft-ietf-quic-multipath.md

```diff
@@ -1274,6 +1274,18 @@ are not congestion controlled (see {{Section 7 of QUIC-RECOVERY}}),
 senders should carefully consider the load induced
 by these packets, especially if the capacity is unknown on that path.
 
+There are various ways that multipath techniques can be used (see {{Section 7.10 of rfc9743}}). 
```

Comment:
Note we already added a reference to RFC9743 in section 5.3 on congestion control:

"Designers of congestion control algorithms specialized for Multipath QUIC are advised to follow BCP 133; see [Section 7.10](https://rfc-editor.org/rfc/rfc9743#section-7.10) of [[RFC9743](https://quicwg.org/multipath/draft-ietf-quic-multipath.html#RFC9743)].[¶](https://quicwg.org/multipath/draft-ietf-quic-multipath.html#section-5.3-2)"

Reply:
Yes, but please make sure somewhere you differentiate between path failover and concurrent/simultaneous usage.

Reply:
RFC9743 does that - explaining the difference between path failover and concurrent use. Why do we need to repeat this here if we don't give any additional advise?